src/hexo/source/_data/contributors.yml

```diff
@@ -1,45 +1,60 @@
 - name: Antón Molleda
   company: Microsoft
   avatar: https://avatars.githubusercontent.com/molant
+  website: https://github.com/molant
```

Comment:
I thought about that, but in case someone wanted to put their personal website/twitter etc.